api/core/model_runtime/model_providers/xinference/llm/llm.py

```diff
@@ -411,6 +411,7 @@ def get_customizable_model_schema(self, model: str, credentials: dict) -> AIMode
         support_function_call = credentials.get('support_function_call', False)
         if support_function_call:
             features.append(ModelFeature.TOOL_CALL)
+            features.append(ModelFeature.STREAM_TOOL_CALL)
```

Comment:
This suggests create a separate PR. and STREAM_TOOL_CALL is more than just adding a feature, it needs to be handled separately for the tool call streaming output.

Reply:
I have reverted this code and opened a new issue: https://github.com/langgenius/dify/issues/5954. I will submit a PR once this is resolved.